src/tests/mockData.js

```diff
@@ -1,53 +1,76 @@
 // Test user for sign-up functionality
-export const newUser = {
+const newUser = {
   first_name: 'Emma',
   last_name: 'Korede',
   email: 'emma.k@yahoo.com',
   password: 'testing123',
 };
 
 // Test Users for Login functionality and other tests that requires a logged in user
-export const testUser1 = {
+const testUser1 = {
   email: 'nancngo@gmail.com',
   password: 'testing123',
 };
 
-export const testUser2 = {
+const testUser2 = {
   email: 'petchidi@yahoo.com',
   password: 'testing123',
 };
 
-export const testManager1 = {
+const testManager1 = {
   email: 'rose.querty@yahoo.com',
   password: 'testing123',
 };
 
-export const testManager2 = {
+const testManager2 = {
   email: 'm.benchfort@yahoo.com',
   password: 'testing123',
 };
 
 // Test Travel Request
-export const testTravelRequest = {
+const testTravelRequest = {
   origin: 'Lagos',
   destination: 'Kigali',
   departure_date: '2019-12-12',
   travel_purpose: 'This is a one way trip',
   accommodation_id: 1653453,
 };
-export const validTravelId = '1891029';
+const validTravelId = '1891029';
 
 // Test comment
-export const testComment = {
+const testComment = {
   comment: 'sample comment on travel request for testing',
 };
 
-export const validCommentId = '554455';
+const validCommentId = '554455';
 
-export const travelRequest = {
+const travelRequest = {
   origin: 'Lagos',
   destination: 'Kigali',
   departure_date: '2019-12-12',
   travel_purpose: 'This is a one way trip',
   accommodation_id: 1653453
 };
+
+const approvedRequest = {
+  origin: 'Lagos',
+  destination: 'Bauchi',
+  departure_date: '2019-12-12',
+  travel_purpose: 'This is a one way trip',
+  accommodation_id: 1653453,
+  approval_status: 'accepted'
+};
+
+export {
```

Comment:
Please revert this to it's former state in this file and in others where you implemented it. The former looks more elegant with way more lesser lines of code. Thanks to our TTL who gave us the suggestion.

Reply:
@chuksjoe  Oh! thanks for the hint

I feel this is the proper way to use export, since it makes it clear what files you are exporting but i will inform the TTL.